iceberg/iceberg-handler/src/test/queries/positive/truncate_force_iceberg_table.q

```diff
@@ -0,0 +1,20 @@
+set hive.vectorized.execution.enabled=false;
+
+drop table if exists test_truncate;
+create external table test_truncate (id int, value string) stored by iceberg stored as parquet;
+alter table test_truncate set tblproperties('external.table.purge'='false');
+insert into test_truncate values (1, 'one'),(2,'two'),(3,'three'),(4,'four'),(5,'five'); 
+insert into test_truncate values (6, 'six'), (7, 'seven');
+insert into test_truncate values (8, 'eight'), (9, 'nine'), (10, 'ten');
+analyze table test_truncate compute statistics;
+
+select * from test_truncate order by id;
```

Comment:
Question: I remember that once upon a time we replaced all of the `order by` statements in the queries to some QTestUtil construct to save on the execution time of the queries. Do I remember correctly? Would it be relevant here? If so, I could try to remember what was needed in the `.q` file to short the output.

Reply:
\oh yeah, you're right. I always forget it. I changed the tests.